blog-cse/2023-11-01-application-update.md

```diff
@@ -0,0 +1,36 @@
+---
+title: November 1, 2023 - Application Update
+keywords:
+  - cloud siem
+  - multi-record rules
+  - automation service
+  - audit logging
+tags: [application update]
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+### Multi-Record Signal Changes
+
+To improve the usability of the Signals user interface, the way that records are displayed on Signals generated by multi-record (Threshold, Chain, and Aggregation) Rules has changed. Instead of attaching a sample set of records to the Signal and then providing a Queried Record tab to manually search for additional records, all records that were part of the Signal will be displayed in the UI. (As a result, the Queried Records tab has been removed from the UI.)
```

Comment:
active voice 

```suggestion
To improve the usability of the Signals user interface, we've changed the way that records are displayed on Signals generated by multi-record (Threshold, Chain, and Aggregation) Rules. Instead of attaching a sample set of records to the Signal and then providing a Queried Record tab to manually search for additional records, all records that were part of the Signal will be displayed in the UI. (As a result, the Queried Records tab has been removed from the UI.)
```